content/en/docs/contribute/participate/pr-wranglers.md

```diff
@@ -84,7 +84,7 @@ To close a pull request, leave a `/close` comment on the PR.
 
 {{< note >}}
 
-The [`fejta-bot`](https://github.com/fejta-bot) bot marks issues as stale after 90 days of inactivity. After 30 more days it marks issues as rotten and closes them.  PR wranglers should close issues after 14-30 days of inactivity.
+The [`k8s-ci-robot`](https://github.com/k8s-ci-robot) bot marks issues as stale after 90 days of inactivity. After 30 more days it marks issues as rotten and closes them.  PR wranglers should close issues after 14-30 days of inactivity.
```

Comment:
It is the `k8s-triage-bot` that marks a PR as 'stale' or 'rotten', `k8s-ci-bot` is the final guardian that changes the PR status. In this context, we are talking about the `fejta-bot` equivalent, which is actually `k8s-triage-bot`.

Reply:
So I should mention _both_ bots in this section? Sounds like I'm headed down a rabbit holes. I was just going by what bot adds the label and making a quick change. 2 of the 5 total PRs I opened have merged, should I be going back and fixing all the wording there?

Reply:
I'd suggest we make this accurate to avoid confusion. Maybe we can just fix the English version, and let the localization team to resync changes like this at their own pace?

Reply:
Sounds good, updated